docassemble/MAVirtualCourt/data/questions/basic_questions_tests.yml

```diff
@@ -20,12 +20,6 @@ code: |
   other_parties.gather()
   allowed_courts = [
     'District Court',
-    'Boston Municipal Court',
```

Comment:
Is it worth changing this back? We might want to test this, but do we also want to test the full list of courts? Should we write a separate test file for testing other starting conditions, etc?